department-of-veterans-affairs/va.gov-cms
# VACMS-6305: Hide paths field for non-admins.

## Description
Relates to #6305

## Testing done
Visual

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/131841281-b4d530ef-1e06-4d53-8373-b0b1985a7690.png)

## QA steps
 - Not sure there are any non-content admin folks who can access banners, yet - the test for this would be to login as a non-content_admin (like Ryan.Stubblebine@va.gov), and go to `/node/33217/edit` & `/node/add/banner` & visually verify the paths field is not present. I tested locally by making the reverse condition true when logged in as an admin.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
